OpenAdaptAI/OpenAdapt
# Alert user when any exception occurs, via sentry hooks

**What kind of change does this PR introduce?**
This PR displays an alert ot the user when an unhandled exception occurs.



**Summary**
When an unhandled exception occurs, an alert box is shown to the user, with a link to join the discord server, or contact openadapt through email.




**Checklist**

**How can your code be run and tested?**
Raise any exception at any part of the code, for example, the `start_recording` function in `tray.py`, and click on the `Record` option in the tray. You should see a window pop up.




**Other information**
